lib/client.js

```diff
@@ -0,0 +1,98 @@
+"use strict";
+
+const Promise = require("bluebird");
```

Comment:
Yeah, I'm well aware of that, in the context I'm talking from (parse-server) we try to not embed those kind of optimizations and leave it for the users to 'opt-in' to reduce the dependencies surface. Kinda show stopper for me, but I may wrap my head around it.